kolibri/core/assets/src/views/CorePage/AppBarPage.vue

```diff
@@ -150,13 +166,30 @@
       this.$nextTick(() => {
         this.appBarHeight = this.$refs.appBar.$el.scrollHeight || 0;
       });
+      window.addEventListener('scroll', this.handleScroll);
```

Comment:
I think there's no need to attach event listener when it's not an app in touch mode, so for performance reasons I'd suggest it's conditioned in that way.

Reply:
And in cases the listener is attached, let's optimize it's handler performance wise. There are more ways to do so, I'd recommend using https://www.npmjs.com/package/frame-throttle (we use this library in KDS as well)